static/app/utils/replays/getCurrentUrl.tsx

```diff
@@ -1,22 +1,23 @@
 import last from 'lodash/last';
 
-import {transformCrumbs} from 'sentry/components/events/interfaces/breadcrumbs/utils';
+import type {Crumb} from 'sentry/types/breadcrumbs';
 import {BreadcrumbType, BreadcrumbTypeNavigation} from 'sentry/types/breadcrumbs';
+import type {EventTransaction} from 'sentry/types/event';
 import {EventTag} from 'sentry/types/event';
-import type ReplayReader from 'sentry/utils/replays/replayReader';
 
 function findUrlTag(tags: EventTag[]) {
   return tags.find(tag => tag.key === 'url');
 }
 
-function getCurrentUrl(replay: ReplayReader, currentOffsetMS: number) {
-  const event = replay.getEvent();
-  const crumbs = replay.getRawCrumbs();
-
+function getCurrentUrl(
+  event: EventTransaction,
+  crumbs: Crumb[],
+  currentOffsetMS: number
+) {
   const startTimestampMs = event.startTimestamp * 1000;
   const currentTimeMs = startTimestampMs + Math.floor(currentOffsetMS);
 
-  const navigationCrumbs = transformCrumbs(crumbs).filter(
```

Comment:
This isn't needed anymore because it's already called as part of `replay.getRawCrumbs()`